packages/core/config/webpack.common.js

```diff
@@ -73,7 +73,7 @@ const getStyleLoaders = (isEnvProduction) => {
   // at web/config/postcss.config.js
   const postCssLoader = hasPostCssConfig
     ? {
-        loader: 'postcss-loader',
+        loader: require.resolve('postcss-loader'),
```

Comment:
@dac09 `postcss-loader` is also optional—users have to have it installed, and we install it as a part of the setup tailwind command